blpapi.ts

```diff
@@ -0,0 +1,640 @@
+/// <reference path="./typings/tsd.d.ts" />
+
+'use strict';
+
+import assert = require('assert');
+import events = require('events');
+import util = require('util');
+
+import Promise = require('bluebird');
+import debug = require('debug');
+import _ = require('lodash');
+
+import path = require('path');
+/* tslint:disable:no-var-requires */
+// TODO REVIEW: Not sure how this module can be imported in a
+// 'TypeScript' fashion.
+var blpapijs = require(path.join(__dirname, '/build/Release/blpapijs'));
```

Comment:
Unfortunately I am observing the same behavior from tsc and tslint when using `typescript-require`. i.e.  a typescript `import` doesn't accept an argument that is a not a string literal and tslint complains when using `require` without `import`.